examples/with-apollo-auth/pages/create-account.js

```diff
@@ -18,11 +18,10 @@ const CreateAccountPage = () => (
 )
 
 CreateAccountPage.getInitialProps = async context => {
-  const { loggedInUser } = await checkLoggedIn(context.apolloClient)
+  const data = await checkLoggedIn(context.apolloClient)
 
-  if (loggedInUser.user) {
-    // Already signed in? No need to continue.
-    // Throw them back to the main page
+  if (data.me) {
+    // If not signed in, send them somewhere more useful
```

Comment:
It looks like registering accounts isn't working 
<img width="1191" alt="Screen Shot 2019-11-10 at 23 41 44" src="https://user-images.githubusercontent.com/22380829/68569766-ce48f900-0413-11ea-891f-d2344b589d23.png">